orchestra/models.py

```diff
@@ -10,7 +11,6 @@
 from orchestra.workflow import Step
 from orchestra.utils.assignment_snapshots import load_snapshots
 
-
 # TODO(marcua): Convert ManyToManyFields to django-hstore referencefields or
 # wait for django-postgres ArrayFields in Django 1.8.
```

Comment:
eh, as long as you know what it means. But a lot of our ManyToManyFields are legitimate, right? (e.g. the self-reference in Certifications.required_certifications).